src/responsive-element.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+var assignBreakpointClasses = function(parent, breakpoints) {
+	var parentWidth = parent.offsetWidth;
+		
+	for (var x=0, l=breakpoints.length; x < l; x++) {
+		if (parentWidth < breakpoints[x].width) {			
+			parent.className += ' ' + breakpoints[x].class;
+		}
+	}
+};
+
+module.exports = function(parent, breakpoints) { // TO DO improve bp handling to recheck on resize (but effeciently...) into own module

```

Comment:
Can we not just export assignBreakpointClasses? Or do we have plans for this function